gcp-observability/src/test/java/io/grpc/gcp/observability/interceptors/LogHelperTest.java

```diff
@@ -645,6 +645,24 @@ public void logRpcMessage() {
               .setLogger(EventLogger.SERVER)
               .build());
     }
+    // message is not of type : com.google.protobuf.Message or byte[]
+    {
+      logHelper.logRpcMessage(
+          seqId,
+          serviceName,
+          methodName,
+          authority,
+          EventType.CLIENT_MESSAGE,
+          new String("message"),
```

Comment:
Why `new String("message")` and not simply `"message"` ?

Reply:
Updated